src/main/resources/divolte.js

```diff
@@ -183,7 +183,7 @@ var AUTO_PAGE_VIEW_EVENT = true;
         // Assumes cookie name and value are sensible. (For our use they are.)
         // Note: No domain means these are always first-party cookies.
         var cookieString = name + '=' + value + "; path=/; expires=" + expiry.toUTCString() + "; max-age=" + maxAgeSeconds;
-        if (domain) {
+        if (domain && domain.length) {
```

Comment:
On `master`, the [Closure Compiler](https://developers.google.com/closure/compiler/) produces identical JavaScript at runtime before and after this change, so I'm not sure we fully understand the problem we're trying to fix.

(Over on #121 I think I've figured out what might be going on and described this in a comment.)